### ISSUE TITLE
Give drop down buttons a more descriptive label

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#930]
**URL:** http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/dac2020/1-2-1/

**Screen shot:**
<img src="https://user-images.githubusercontent.com/43404358/94606693-c051d000-0292-11eb-92c4-399013f08f49.png" height="350px">

![image](https://user-images.githubusercontent.com/43404358/94606704-c5168400-0292-11eb-9600-a8b6949da40a.png)

The buttons to show sub-categories are marked with an aria-expanded, but the purpose of the components was not determinable from the label to screen reader users navigating ‘out of context’, filtering the page by element type.

**Current Code Ref(s):** #fields > fieldset:nth-child(1) > div > button

```
<button class="accessBtn" tabindex="0" aria-expanded="false">
  <h5>Sex<i class="fa fa-chevron-down"></i></h5>
</button>
```

**Solution:**
Please consider a more descriptive label.

**Example:**
```
<button class="accessBtn" tabindex="0" aria-expanded="false">
  <h5>
    <span aria-hidden="true">Sex<i class="fa fa-chevron-down"></i></span>
    <span class="sr-only">Show Sub-Categories: Sex</span>
  </h5>
</button>
```

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning Reopening as I sent this to DAC for sign off and had the following reply:

> this still has a label ‘press to hide’ when it is pressed so it should be ‘press to show’:
> 
> ```
> <button aria-pressed="true" aria-label="Press to hide the data set By 5th birthday, Measles mumps rubella (first and second dose)"><span class="swatch" style="background-color: #7e984f"></span>By 5th birthday, Measles mumps rubella (first and second dose)</button>
> ```

http://sdgdev-813006012.eu-west-1.elb.amazonaws.com/more-desc-button-label/3-b-1/

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin I'm having trouble finding this "Press to hide the data set" text in our code. Can you describe where exactly this button is appearing?

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin Aha, I see now that it is in the clickable chart legend items. Since those are no longer clickable, I think we can close this one. Do you agree?

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning ah yes, you're right - I originally thought they were referring to something in the sidebar!
